data-reporting-service/investigation-service/Dockerfile

```diff
@@ -1,14 +1,12 @@
 FROM amazoncorretto:17 as builder
 #Copy sources
-COPY data-reporting-service/investigation-service /usr/src/data-reporting-service/investigation-service
-COPY gradle /usr/src/data-reporting-service/gradle
-COPY gradlew /usr/src/data-reporting-service/gradlew
+COPY data-reporting-service /usr/src/data-reporting-service
```

Comment:
Is there a reason to remove the copy config files here? and also, do we not need to copy the `common-util` library, so that the latest code would be built all the time?

Reply:
I am copying source code of all the folders under `data-reporting-service` which will include `common-util`
So we will have the latest code everytime.

Reply:
But if we're building only the individual services, why to copy every other services from the source folder?